bot/exts/discord/discord.py

```diff
@@ -74,16 +76,30 @@ async def embed(
                 raise BadArgument(str(e))
 
         send_disclaimer = not await self.bot.is_owner(inter.author)
-        try:
-            await inter.response.send_message(
-                EMBED_WARNING if send_disclaimer else "",
-                embed=embed,
-            )
-        except commands.CommandInvokeError:
-            raise commands.BadArgument(
-                "No valid embed could be generated form the given input."
-            )
+        await inter.response.send_message(
+            EMBED_WARNING if send_disclaimer else "",
+            embed=embed,
+        )
 
+    @embed.error
+    async def on_embed_error(
+        inter: ApplicationCommandInteraction,
+        error: commands.CommandInvokeError,
+    ) -> None:
+        """Error event of command `embed`"""
+        match error.original:
+            # https://stackoverflow.com/a/67525259/13884898
+            # It's required to do `disnake.HTTPException`,
+            # instead of just `HTTPException`
+            case disnake.HTTPException:
+                raise commands.BadArgument(
+                    "Invalid URL responded."
```

Comment:
```suggestion
                    "Invalid URL provided."
```
Maybe something like this? Or could we be more specific, like what attribute is creating this issue?